src/popup/components/AccountInfo.vue

```diff
@@ -68,6 +69,7 @@ export default defineComponent({
     idx: { type: Number, default: 0 },
     canCopyAddress: Boolean,
     isMultisig: Boolean,
+    avatarWithoutBorder: Boolean,
```

Comment:
As the `Avatar` component is using `borderless` prop I suggest to rename this prop to `avatarBorderless`.